components/tooltip/tooltip.js

```diff
@@ -449,6 +449,9 @@ class Tooltip extends RtlMixin(LitElement) {
 		this._isHovering = false;
 		this._resizeRunSinceTruncationCheck = false;
 		this._viewportMargin = defaultViewportMargin;
+
+		this.#isHoveringTooltip = false;
+		this.#leftTooltip = false;
```

Comment:
Open to a better name for this

Reply:
Hmmm yeah, I can see it being a little confusing. Idk, maybe `mouseLeftTooltip` would be a bit clearer. Or perhaps `wasHoveringTooltip`, or `mouseLeavingTooltip`?